carma_cooperative_perception/.clang-tidy

```diff
@@ -0,0 +1,15 @@
+---
+Checks: >-
```

Comment:
I think sonarqube does a very similar static analysis. I have also seen there are ways to incorporate external static analysis into sonar cloud report. We may want to check what parts of this static analysis overlap with sonarqubes and see if there is a need for both tools. I know for example sonarqube does static analysis on cppcoreguidelines, performance, cert, bugprone. Its possible sonarqube is using clang-tidy in the background although I am not sure of that. Just a thought. Want to make sure we are not adding any tools for functionality we already have.

Reply:
I was using `clang-tidy` for local static analysis, but I will remove the file so it doesn't inadvertently mess with SonarQube. I would like to integrate `clang-tidy` into our workflow in the near future.